pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -319,7 +324,7 @@ func (vm *windows) initializeBootstrapperFiles() error {
 }
 
 // createService creates the service on the Windows VM
-func (vm *windows) createService(svc service) error {
+func (vm *windows) createService(svc *service) error {
 	out, err := vm.Run("sc.exe create "+svc.Name()+" binPath=\""+svc.BinaryPath()+" "+
```

Comment:
instead of calling the methods here to return the fields, you could grab the struct fields contents here directly, i.e. `svc.name`, `svc.binaryPath`, and `svc.args`